package.json

```diff
@@ -55,7 +55,6 @@
     "@babel/core": "7.17.7",
     "@deephaven/components": "file:../components",
     "@deephaven/console": "file:../console",
-    "@deephaven/eslint-config": "file:../eslint-config",
```

Comment:
I'm a little confused...
- `.eslintrc` references `@deephaven/eslint-config`, does this not need to be a dev dep?
- Why do the `devDependencies` all have `../` instead of `package/` in there value?

Reply:
Oops. I removed this while trying to fix package-lock stuff. I checked out the main package-lock and when running `npm install` after updating the versions, I was getting a bunch of peer dep issues. Turned out I need to `npm install eslint@version -w @deephaven/eslint-config` to get npm to play nice